src/Microsoft.AspNetCore/WebHost.cs

```diff
@@ -162,9 +162,9 @@ public static IWebHostBuilder CreateDefaultBuilder(string[] args)
                         config.AddCommandLine(args);
                     }
                 })
-                .ConfigureLogging(logging =>
+                .ConfigureLogging((context, logging) =>
                 {
-                    logging.AddConsole();
+                    logging.AddConsole(context.Configuration);
```

Comment:
Slight behaviour difference from before but I think it's reasonable to pass the configuration to the console logger by default.